crates/fluvio-spu/src/services/public/api_versions.rs

```diff
@@ -12,13 +12,10 @@ use fluvio_spu_schema::server::update_offset::UpdateOffsetsRequest;
 use fluvio_spu_schema::{ApiVersionsRequest, ApiVersionsResponse};
 
 #[instrument(skip(request))]
-pub async fn handle_kf_lookup_version_request(
+pub async fn handle_api_version_request(
```

Comment:
Let's put this as start of span by removing skip

Reply:
Let's revisit this when we talk about spans in more depth